docs/src/components/PlaygroundCodeEditor.tsx

```diff
@@ -1,33 +1,57 @@
-import styles from './PlaygroundCodeEditor.module.css';
-import { ChangeEvent, useState } from 'react';
+import React, { useState, useEffect, useCallback } from 'react';
+import 'jsoneditor/dist/jsoneditor.css';
+import './jsoneditor-dark.css';
+import componentTypesJsonSchema from '../../../schemas/component_types.schema.json';
```

Comment:
This file probably won't deploy on the website (correct me if I'm wrong). Putting it in `docs/static` would fix it (files from `docs/static/[some_path]` are hosted on `compositor.live/[some_path]` after docs deploy). Also, consider committing this file and adding to Ci check if generated version matches committed one (similarly to `.scene.json` schema)

Reply:
this is a subset of another schema, so I would prefer not commiting that. We already check if that other schema changed.

> This file probably won't deploy on the website (correct me if I'm wrong)
I'm not sure if this won't work, but yes make sure to verify it